crates/solidity/outputs/rust/lib/src/generated/kinds.rs

```diff
@@ -116,17 +123,19 @@ pub enum TokenKind {
     PlusEqual,
     PlusPlus,
     PossiblySeparatedPairsOfHexDigits,
-    Pragma,
-    Private,
-    Public,
-    Pure,
+    PragmaKeyword,
+    PrivateKeyword,
+    PublicKeyword,
+    PureKeyword,
     Question,
     RawIdentifier,
     Receive,
```

Comment:
I think we missed a `Receive` in `crates/solidity/inputs/schema/grammar/topics/02-definitions/08-functions.yml`

Reply:
done